src/screens/AddNewItemScreen/index.tsx

```diff
@@ -36,12 +40,36 @@ export const AddNewItemScreen: FC = () => {
 
   return (
     <View style={style.container}>
-      <TextInput
-        style={style.input}
-        placeholder="New task"
-        onChangeText={setText}
-        value={text}
-        selectionColor={COLORS.black}
+      <View style={{ width: '100%' }}>
+        <TouchableOpacity
+          style={style.input}
+          onPress={() => inputRef.current?.focus()}>
+          <Text>{'Add new task'}</Text>
```

Comment:
you should create separete component that has title, default focus when press on input form

Reply:
ok